vendor/k8s.io/utils/trace/trace.go

```diff
@@ -44,6 +44,25 @@ func writeFields(b *bytes.Buffer, l []Field) {
 	}
 }
 
+func logTraceArgs(traceNum int32, trace *Trace, totalTime time.Duration) []interface{} {
+	result := make([]interface{}, 0, 2*len(trace.fields)+8)
+	result = append(result, "traceNum", traceNum, "traceName", trace.name)
+	for _, field := range trace.fields {
+		result = append(result, field.Key, field.Value)
+	}
+	result = append(result, "startTime", trace.startTime, "totalTime", totalTime)
```

Comment:
Why split append into 2 calls?

Reply:
Fields[] is outputed in the middle in original trace log message:
(tracenum, t.name, Field[], starttime, totaltime)
```
	buffer.WriteString(fmt.Sprintf("Trace[%d]: %q ", tracenum, t.name))
	if len(t.fields) > 0 {
		writeFields(&buffer, t.fields)
		buffer.WriteString(" ")
	}
	buffer.WriteString(fmt.Sprintf("(started: %v) (total time: %v):\n", t.startTime, totalTime))
```

So, I splited in order to keep order of element of trace as the original.
If Field[] is outputtd the end of original log message, we don't need to split.

Reply:
Is there any modification about this?